servo/errors.py

```diff
@@ -72,6 +72,11 @@ def event(self) -> Optional[servo.Event]:
         """The event that was executing when the error occurred."""
         return self._event
 
+    def __str__(self) -> str:
```

Comment:
Does it make sense to implement `repr`? Not sure what this looks like when logged

Reply:
This change was removed per Peter's ticket feedback; sending reason in the message is redundant and should be logged clearly in the backend